src/views/Pools/components/CardFooter.tsx

```diff
@@ -105,6 +105,8 @@ const CardFooter: React.FC<Props> = ({
 
   const imageSrc = `${BASE_URL}/images/tokens/${tokenName.toLowerCase()}.png`
 
+  const isMetaMaskInScope = !!Reflect.get(window, 'ethereum')?.isMetaMask
```

Comment:
I have never heard of [Reflect](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect) before. Is there a benefit to using it?

Reply:
I like `Reflect` object because it's very semantically correct as to what it does - it houses reflective functions.
Those functions also have very straight forward names, such as `get`, `has`, `set`, etc.

In this case it's also in a way a workaround as to not have to do type casting.

Reply:
It's better to stay coherent with the rest of the codebase

Reply:
@RabbitDoge Sure thing!
Hadn't noticed before that there is a `WindowChain` interface I could use.